L/libcgal_julia/build_tarballs.jl

```diff
@@ -4,18 +4,18 @@ using BinaryBuilder
 
 const name = "libcgal_julia"
 
-version = v"0.4.0"
+version = v"0.5.1"
 
 # Collection of sources required to build CGAL
 const sources = [
-    "https://github.com/rgcv/libcgal-julia.git" =>
-        "8d97853ac3baf5fee431d005b23a45e0390a9f81"
+    GitSource("https://github.com/rgcv/libcgal-julia.git",
+              "370a0af377082ed73c57e7392a74645500ba2a0a"),
 ]
 
 # Dependencies that must be installed before this package can be built
 const dependencies = [
-    "CGAL_jll",
-    "libcxxwrap_julia_jll",
+    Dependency("CGAL_jll"),
+    Dependency("libcxxwrap_julia_jll"),
```

Comment:
Is this a build-time or run-time dependency? :thinking:

Reply:
I'm thinking both. Without one or the other, `libcgal_julia` won't build. The binaries are definitely needed at run-time as well. I'm building shared libraries, not static ones.

Reply:
`libcxxwrap_julia` might be considered a build-time dependency, seeing it will be present when dragged in by the `CxxWrap.jl` at run-time. The only glaring build dependency right now is Julia itself, but that's a whole nother ordeal.